### PR TITLE
Chore/add docker setup

### LINKED ISSUE #9: Docker setup

### PR DESCRIPTION
**Pull request by faheelsattar**
Closes #9

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      - KEY_BROADCAST_CONTRACT_ADDRESS=${KEY_BROADCAST_CONTRACT_ADDRESS}
+      - KEYPER_SET_MANAGER_CONTRACT_ADDRESS=${KEYPER_SET_MANAGER_CONTRACT_ADDRESS}
+      - KEYPER_HTTP_URL=${KEYPER_HTTP_URL}
+      - SIGNING_KEY=${SIGNING_KEY}
+      - SERVER_PORT=23003
+    ports:
+      - "23003:23003"
+    command: ["./shutter-service-api"]
+
+  db:
```
Comment (blockchainluffy) [inline docker-compose.yml:28]: why do we need database configured here? we would be using db setup by keyper

**Review by blockchainluffy — CHANGES_REQUESTED** [review]
added some comments

File: docker-compose.yml (hunk 1 above)
Comment (faheelsattar) [reply docker-compose.yml:28]: added for testing purposes

**Review by blockchainluffy — APPROVED** [review]

**Comment by blockchainluffy** [comment]
we would need to update the compose with exact setup when database is initialised via keypers and remove db from compose here

**Pull request merged**